lib/notesclub/github_api.ex

```diff
@@ -66,6 +104,26 @@ defmodule Notesclub.GithubAPI do
     end
   end
 
+  defp extract_user_info(%GithubAPI{response: response, errors: errors} = fetch) do
+    with false <- errors[:github_api_key] == ["is missing"],
+         200 <- response.status do
+      user_info = %{
+        github_twitter_username: response.body["twitter_username"],
+        github_real_name: response.body["name"]
+      }
```

Comment:
```suggestion
      user_info = %{
        twitter_username: response.body["twitter_username"],
        name: response.body["name"]
      }
```

If we return this, it will be easier to call `Accounts.update_user(user, user_info)`.

Reply:
Yep, agreed! Updated.